Sources/Common/Core/MatrixBuilder/index.js

```diff
@@ -128,6 +130,11 @@ class Transform {
     }
     return this;
   }
+
+  identity() {
+    mat4.identity(this.matrix);
```

Comment:
Is this an opengl matrix (column-major) ?
We discussed with @martinken and @thewtex, and we'll probably want to switch matrix library to consolidate our math implementation in vtk-js (itk, vtk, itk.js, vtk.js Math are row-major, but opengl is column-major, which lead to a lot of confusions).

Reply:
Correct, that was more a global comment regarding the matrix in vtkMatrixBuilder. How should I fill it up in setMatrix?

Reply:
Based on your comment, I would have 2 methods which encode the matrix format in their name or via an argument flag.